spyder/plugins/console/widgets/internalshell.py

```diff
@@ -129,34 +129,30 @@ def end_input(self, cmd):
 class InternalShell(PythonShellWidget):
     """Shell base widget: link between PythonShellWidget and Interpreter"""
 
-    status = Signal(str)
-    refresh = Signal()
-    go_to_error = Signal(str)
-    focus_changed = Signal()
+    sig_refreshed = Signal()
+    sig_show_status_requested = Signal(str)
+    sig_go_to_error_requested = Signal(str)
+    sig_focus_changed = Signal()
 
     def __init__(self, parent=None, namespace=None, commands=[], message=None,
-                 max_line_count=300, font=None, exitfunc=None, profile=False,
+                 max_line_count=300, exitfunc=None, profile=False,
                  multithreaded=True):
-        PythonShellWidget.__init__(self, parent,
-                                   get_conf_path('history_internal.py'),
-                                   profile=profile)
+        super().__init__(parent, get_conf_path('history_internal.py'),
+                         profile=profile)
 
         self.multithreaded = multithreaded
         self.setMaximumBlockCount(max_line_count)
 
-        if font is not None:
-            self.set_font(font)
-
         # Allow raw_input support:
         self.input_loop = None
         self.input_mode = False
 
         # KeyboardInterrupt support
-        self.interrupted = False # used only for not-multithreaded mode
+        self.interrupted = False  # used only for not-multithreaded mode
         self.sig_keyboard_interrupt.connect(self.keyboard_interrupt)
 
         # Code completion / calltips
-        getcfg = lambda option: CONF.get('internal_console', option)
+        # getcfg = lambda option: CONF.get('internal_console', option)

```

Comment:
Remove this comment if it's no longer necessary.

Reply:
Fixed